.dockerignore

```diff
@@ -0,0 +1,3 @@
+.git
+.gitignore
+k8s
```

Comment:
I'd prefer if the directory was named `kubernetes` since maybe all developers/users are not aware of the shorthand.